tools/clang-archer++.in

```diff
@@ -68,4 +68,4 @@ else
   link_flags=""
 fi
 
-@LLVM_ROOT@/bin/clang++ -I@OMP_PREFIX@/include -Xclang -load -Xclang @CMAKE_INSTALL_PREFIX@/lib/LLVMArcher.so -fopenmp -fsanitize=thread $link_flags -g "$@"
+@LLVM_ROOT@/bin/clang++ -I@OMP_PREFIX@/include -Xclang -load -Xclang @CMAKE_INSTALL_PREFIX@/lib/LLVMArcher.so -std=c++11 -fopenmp -fsanitize=thread $link_flags -g "$@"
```

Comment:
For every c++ program that I am compiling I am getting compiler errors, and I always have to add "-std=c++11", but yeah I can't generalize, I'll revert it.